### DISCUSSION TITLE
How to use maximal_ball_wrapper? Can anybody show this with an example with visualization.

### DISCUSSION THREAD

**Discussion by PratyushSwain101** [PMEAL/porespy#904] [Q&A]
I want to use the maximum ball algorithim to my micro CT data , can anybody help me with maximal_ball_wrapper, how to use it.

**Comment by jgostick** [comment]
We don't support that anymore and will be removing it in V3.  Just find the binary on Martin Blunts website and use that directly.

**Reply by PratyushSwain101** [reply]
Can you please provide me the link?

**Comment by jgostick** [comment]
google.com?

**Reply by PratyushSwain101** [reply]
yes
